big_tests/tests/gdpr_SUITE.erl

```diff
@@ -194,6 +249,26 @@ inbox_opts() ->
      {groupchat, [muclight]},
      {markers, [displayed]}].
 
+inbox_opts(muc) ->
+    [{aff_changes, true},
+     {remove_on_kicked, true},
+     {groupchat, [muc]},
+     {markers, [displayed]}].
+
+
+prepare_room(Owner, Members, Config) ->
```

Comment:
`prepare_muclight_room` ? as we are dealing with two types for MUCs in this test